simpeg/electromagnetics/static/resistivity/survey.py

```diff
@@ -51,10 +51,20 @@ def survey_geometry(self):
         str
             Survey geometry; one of {"surface", "borehole", "general"}
         """
+        warnings.warn(
+            "Argument 'survey_geometry' is deprecated and will be removed in SimPEG 0.25.0.",
+            FutureWarning,
+            stacklevel=2,
+        )
```

Comment:
Moreover, this warning is stating that we'll be removing it in 0.25.0. If we are deprecating it, we should schedule its removal on 0.26.0 as minimum.